### PR TITLE
Campaign -- Citizen visualization

### PR DESCRIPTION
**Pull request by amy-lei**
Just like with budget simulator, there is now a component to represent a sample of the population to provide context/info about the type of people that voted:
- Fill color indicates whether the citizen supports
- Hovering over the svg will produce a popup with name and preferences
- Responsive to changes

image
image

### CODE REVIEW

File: frontend/src/campaign/campaignView.js [hunk 1]
```diff
@@ -214,6 +216,62 @@ Results.propTypes = {
     countryName: PropTypes.string,
 };
 
+class Citizen extends React.Component {
```
Comment (ryaanahmed) [inline frontend/src/campaign/campaignView.js:219]: Is this copy/paste from the budget sim? For the sake of expediency right now, I'm okay with that for now, but we should properly factor this out later.

**Review by ryaanahmed — APPROVED** [review]
looks good -- merging, so we can get on to making this the "survey data" for the hints.

File: frontend/src/campaign/campaignView.js (hunk 1 above)
Comment (amy-lei) [reply frontend/src/campaign/campaignView.js:219]: Yeah I can factor this out when we use it for feedback later

Comment (ryaanahmed) [reply frontend/src/campaign/campaignView.js:219]: sounds good!

**Pull request merged**